packages/@lwc/template-compiler/src/__tests__/fixtures/regression/deprecated-void-elements/keygen/metadata.json

```diff
@@ -0,0 +1,48 @@
+{
+    "warnings": [
+        {
+            "code": 1057,
+            "message": "LWC1057: name is not valid attribute for keygen. For more information refer to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/keygen",
```

Comment:
this is supported by keygen

Reply:
Same thing as the above except for `keygen`